### ISSUE TITLE
Command-line slicing only slices first filename

### ISSUE THREAD

**Issue by thinkyhead** [daid/LegacyCura#482]
Regardless of how many filenames are given, only the first file is sliced. Is the new command-line specification to only slice a single file at a time, or is this a bug?

I added Cura support to https://github.com/thinkyhead/Slic3r-Droplet-for-OSX but I find I have to keep adding more conditionals to deal with recent changes.

**Comment by daid** [comment]
Should be a few easy fixes, combined with the .stl.gcode problem you had. Just let me know what you need.

**Comment by thinkyhead** [comment]
I haven't tested much with 13.06.5 yet, so I'll give it a run-through in the next day or two. In the long run I just need the command line to work predictably. I don't mind if it's completely different from Slic3r's. For yours I have assumed from the --help output that I should use...

```
Cura.app/Contents/MacOS/Cura [-i initfile.ini] [-s file1.stl [file2.stl […]]]
```

...and with Cura 12 this worked, producing output files named file1.gcode and file2.gcode, for example. That behavior seems fine, especially since with multiple inputs specified, Cura could theoretically divide slicing between cores, or optimize in other ways. With Cura 13 only a single file is sliced, which due to your much faster slicing engine is not as bad as it might be.

The only helpful option Slic3r includes which Cura lacks is the output file name. In terms of how my AppleScript droplet works, the final output can optionally be renamed with any extension, such as .gco or .g. In the Slic3r codepath Slic3r can produce the extension directly. In the Cura codepath there's just an additional step, renaming the output file(s).

I haven't added support for other slicing software yet, but I've got that on my list. Although I'm currently just playing around with this idea as a droplet, it may develop into something more comprehensive for organizing STL files, configurations, and slicers. I'm not sure how many people work like I do, where they keep sets of STL files around, occasionally printing them for people and projects, and always putting them onto SD cards, which I just find faster and more reliable with RAMPS 1.4.

I've automated the syncing of files from a "Card" folder on my laptop to an SD Card whenever I insert it writeable, but each time I get new slicers or change settings I prefer to re-generate all the .gcode files in a batch using a few different .ini files, one for coarse ABS, one for fine ABS, others for PLA. The gcode files have the name of the config file appended, so I can see at a glance what they are. As automated as that is, it still means I have to go to the file system to drag the files onto the droplets, then rename, move, etc. I doubt I'm the only person trying to manage all these files and wishing there was a "one button" solution.

Anyway, I'll continue to play around with Cura and see where it's at.
